Makefile

```diff
@@ -27,10 +27,13 @@ CHART_VERSION ?= local
 
 HELM_UNITTEST_IMAGE ?= quintush/helm-unittest:3.3.0-0.2.5
 
-all : test chart
+all : check_policy_whitespace test chart
 test: chart_test rego_test
 chart: chart_setup chart_package
 
+check_policy_whitespace:
+		@echo 'Testing Policy for Whitespace'; sh -c "grep -ne '[[:space:]]$$' kubernetes/cray-opa/templates/_policy*.tpl && exit 1; exit 0"; exit $$?
```

Comment:
We're just looking for trailing whitespace as a failure case correct? If so easier to use awk for this than to do a subshell with make and &&.

```
awk '/[[:space:]]+$/ { exit 1 }' files
```

Also nice to not stick it all on a single @ line so make V=1 shows the command thats failing.

If we need the line number of the match can add that into the awk too, something like:

```
$ awk '/[[:space:]]+$/ { count++; print FILENAME, "@", NR, ":", $0; } END{ exit count }' *.txt
foo.txt @ 2 : i have a space at the end 
foo.txt @ 4 : I have a tab at the end
zsh: exit 2     awk  *.txt
```

But I might be off on whats happening.

Reply:
Thanks @mitchty. I don't find that it's easier, in a quick effort, to use awk directly in a Makefile, without a sub-shell. That's not to say it's not possible. I also didn't attempt to use (extend) awk to match your mockup to print the actual character(s) matched from the pattern, e.g., 'space', 'tab' (although this would be nice). 

In terms of output, we do get the lines impacted with the existing mod, at least using grep on Mac:

```
jeremyd@Jeremys-MacBook-Pro cray-opa % make check_policy_whitespace
Testing Policy for Whitespace
kubernetes/cray-opa/templates/_policy-ingressgateway.tpl:81:  http_request.method in {"GET", "HEAD"}  
make: *** [check_policy_whitespace] Error 1
```

May revisit this, but need to get this mod in for the 1.3.1 patch.